lib/parsers/docs-parser.ts

```diff
@@ -51,7 +51,7 @@ export async function parseFile(file: IParseFile, ids: Record<string, string>) {
   }
 
   file.sections = await Promise.all(
-    splitMd(await fixMdLinks(markdown)).map(async (section: $TSFixMe) => {
+    splitMd(await fixMdLinks(markdown)).map(async (section: Section) => {
```

Comment:
```suggestion
    splitMd(await fixMdLinks(markdown)).map(async (section: ISection) => {
```

Reply:
Just curious don't here `ISection` needed? TS should automatically give a type for the section as `ISection`